src/layouts/Footer/Footer.spec.tsx

```diff
@@ -71,14 +56,8 @@ describe('Footer', () => {
         const pricing = screen.getByText('Pricing')
         expect(pricing).toBeInTheDocument()
       })
-
-      it('renders licensing link', () => {
```

Comment:
Per @aj-codecov, we are removing the Licensing link for Enterprise customers that formerly went to broken link at https://docs.codecov.com/docs/self-hosted-dependency-licensing